tests/global_config_s390x.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any
+
+import pytest_testconfig
+
+import utilities.constants
+from utilities.constants import (
+    EXPECTED_CLUSTER_INSTANCE_TYPE_LABELS,
+    PREFERENCE_STR,
+    S390X,
+    Images,
+)
+
+Images.Fedora.FEDORA41_IMG = "Fedora-Cloud-Base-Generic-41-1.4.s390x.qcow2"
+Images.Rhel.RHEL9_5_IMG = "rhel-95-s390x.qcow2"
+Images.Fedora.FEDORA_CONTAINER_IMAGE = "quay.io/chandramerla/qe-cnv-tests-fedora:40-s390x"
+EXPECTED_CLUSTER_INSTANCE_TYPE_LABELS[PREFERENCE_STR] = f"rhel.9.{S390X}"
+NET_UTIL_CONTAINER_IMAGE = "quay.io/chandramerla/qe-cnv-tests-net-util-container:centos-stream-9"
+
+# No support for cirros on s390x.  Use Fedora instead
+Images.Cirros.RAW_IMG = "Fedora-Cloud-Base-Generic-41-1.4.s390x.raw"
+Images.Cirros.RAW_IMG_GZ = "Fedora-Cloud-Base-Generic-41-1.4.s390x.raw.gz"
+Images.Cirros.RAW_IMG_XZ = "Fedora-Cloud-Base-Generic-41-1.4.s390x.raw.xz"
+Images.Cirros.QCOW2_IMG = "Fedora-Cloud-Base-Generic-41-1.4.s390x.qcow2"
+Images.Cirros.QCOW2_IMG_GZ = "Fedora-Cloud-Base-Generic-41-1.4.s390x.qcow2.gz"
+Images.Cirros.QCOW2_IMG_XZ = "Fedora-Cloud-Base-Generic-41-1.4.s390x.qcow2.xz"
+Images.Cirros.DIR = Images.Fedora.DIR
+Images.Cirros.DEFAULT_DV_SIZE = Images.Fedora.DEFAULT_DV_SIZE
+Images.Cirros.DEFAULT_MEMORY_SIZE = Images.Fedora.DEFAULT_MEMORY_SIZE
```

Comment:
since you're replacing all Cirros constants, how about smth like this
```
class Cirros:
    ...

Images.Cirros = Cirros
```

Reply:
done.  also Cdi.